### PR TITLE
bug:[CORE-1333] correct datatype for createdDate and modifiedDate in …

### PR DESCRIPTION
**Pull request by shefalibisht1992**
…asset groups

# Description

Datatype for asset group createdDate and modifiedDate is varchar in database, correctting it to timestamp.
ag1
aset3
asset-grp1
assetgrp2

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

**Review by plyubich — CHANGES_REQUESTED** [review]
Please correct formatting of the code

**Comment by shefalibisht1992** [comment]
> Please correct formatting of the code

done.

**Review by santhosh-challa — APPROVED** [review]

**Review by plyubich — CHANGES_REQUESTED** [review]
Please work on formatting

File: installer/resources/pacbot_app/files/DB.sql [hunk 1]
```diff
@@ -3186,3 +3186,80 @@ END //
...
+END loop myloop;
+
+ALTER TABLE cf_AssetGroupDetails modify COLUMN createdDate TIMESTAMP DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE cf_AssetGroupDetails modify COLUMN modifiedDate TIMESTAMP DEFAULT CURRENT_TIMESTAMP;
+CLOSE dateCursor;
+END IF;
+END //
+DELIMITER ;
+
+CALL correct_datatype_for_dates_in_asset_groups();
```
[the remark this answers is not in the document]
Comment (shefalibisht1992) [reply installer/resources/pacbot_app/files/DB.sql]: Done.

**Review by santhosh-challa — APPROVED** [review]

**Review by raviteja-mandala — APPROVED** [review]

**Pull request merged**